lib/drivers/mock/mock_test.go

```diff
@@ -1 +1,80 @@
 package mock
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/PacketFire/immigrant/lib/core"
+)
+
+func TestMockDriver_Init(t *testing.T) {
+	type args struct {
+		config map[string]string
+	}
+	tests := []struct {
+		name    string
+		this    *MockDriver
+		args    args
+		wantErr bool
+	}{}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if err := tt.this.Init(tt.args.config); (err != nil) != tt.wantErr {
+				t.Errorf("MockDriver.Init() error = %v, wantErr %v", err, tt.wantErr)
+			}
+		})
+	}
+}
+func TestMockDriver_Migrate(t *testing.T) {
+	type args struct {
+		r  *core.Revision
+		ec chan error
+	}
+	tests := []struct {
+		name string
+		this *MockDriver
+		args args
+	}{}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			tt.this.Migrate(tt.args.r, tt.args.ec)
+		})
+	}
+}
+
+func TestMockDriver_Rollback(t *testing.T) {
+	type args struct {
+		r  *core.Revision
+		ec chan error
+	}
+	tests := []struct {
+		name string
+		this *MockDriver
+		args args
+	}{}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			tt.this.Rollback(tt.args.r, tt.args.ec)
```

Comment:
It looks like you're invoking the function we're interested in testing, but (correct me if i'm wrong) you're not asserting that the effects of that function are correct. How can we verify that the rollback operation actually happened?